ufersa/plataforma-sabia
# Remove NextJS custom server

## Summary

Resolves #978 

## Relevant technical choices

- Changed from `next-i18next` to `next-translate` 
- Now our tests receives the translations, so I had to update all snapshots and expects that was using namespaces like `common:no`
- Plural form changed from `_plural` to `_other` due language specifications (see [doc](https://github.com/vinissimus/next-translate#5-plurals))

## QA Steps

1. Access the application
2. Check if all pages works as expected
3. Check if theres no changes in translations files

## Checklist
